spring-integration-azure/src/main/java/com/microsoft/azure/spring/integration/storage/queue/inbound/StorageQueueInboundChannelAdapter.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+
+package com.microsoft.azure.spring.integration.storage.queue.inbound;
+
+
+import com.microsoft.azure.spring.integration.core.QueueOperation;
+import com.microsoft.azure.spring.integration.core.SubscribeOperation;
+import com.microsoft.azure.storage.queue.CloudQueueMessage;
+import org.springframework.integration.endpoint.MessageProducerSupport;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+import org.springframework.messaging.converter.MessageConverter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+public class StorageQueueInboundChannelAdapter extends MessageProducerSupport {
+    private final String destination;
+    private final QueueOperation<CloudQueueMessage> queueOperation;
+    private MessageConverter messageConverter;
+    private Map<String, Object> commonHeaders = new HashMap<>();
+
+    public StorageQueueInboundChannelAdapter(String destination,
+                                             QueueOperation<CloudQueueMessage> queueOperation) {
+        this.destination = destination;
+        this.queueOperation = queueOperation;
+    }
+
+    @Override
+    protected void doStart(){
+        super.doStart();
+        ScheduledThreadPoolExecutor scheduledThreadPoolExecutor = new ScheduledThreadPoolExecutor(1);
+        scheduledThreadPoolExecutor.scheduleAtFixedRate(
+                this::receiveMessage, 10, 10, TimeUnit.SECONDS);
```

Comment:
What is this two `10` numbers here? Does that mean the user who uses this queue cannot change the receive message interval etc.? And if we need to use numbers, can we move these numbers as constant variables, with proper naming what it is? cc @warrenzhu25

Reply:
I suggest to write StorageQueueMessageSource first, which could take advantage of Poller provided by spring integration.